skimage/morphology/tests/test_flood_fill.py

```diff
@@ -93,7 +93,7 @@ def test_inplace_noncontiguous():
     # Transpose is noncontiguous
     image2 = image[::2, ::2]
 
-    flood_fill(image2, (0, 0), 5, inplace=True)
+    flood_fill(image2, (0, 0), 5, in_place=True)
```

Comment:
Can you duplicate one of these tests to ensure that the old parameter still works?

That test can then be easily removed when it comes time to finally rice the behavior. There is also a way to enforce q check on the working being issued.

Reply:
That's a very nice input! Thanks :+1: